### PR TITLE
Reproduced with integration test and fixed in http module

### PR DESCRIPTION
**Pull request by g4s8**
#370 - reproduced HTTP proxy issue with integration test, fixed in http module, and update http version.

### CODE REVIEW

File: src/test/java/com/artipie/http/GroupRepositoryITCase.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+/**
+ * Integration tests for grouped repositories.
+ * @since 0.10
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+@EnabledIfSystemProperty(named = "test.networkEnabled", matches = "true|yes|on|1")
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/http/GroupRepositoryITCase.java:53]: @g4s8 `networkEnabled` is not enabled anywhere, are you planning to do it in another PR? We need a todo then

**Comment by g4s8** [comment]
@olenagerasimova thanks, added todo

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @victornoel[/z](https://www.0crat.com/u/victornoel)[/z](https://www.0crat.com/u/victornoel) didn't make CR comments

**Comment by 0crat** [comment]
The project doesn't have enough funds, can't make a payment
